pkg/pgmodel/cache/series_cache.go

```diff
@@ -2,70 +2,81 @@
 // Please see the included NOTICE for copyright information and
 // LICENSE for a copy of the license.
 
-package model
+package cache
 
 import (
 	"encoding/binary"
 	"fmt"
 	"math"
 	"sort"
 	"strings"
-	"sync"
 
 	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/timescale/promscale/pkg/clockcache"
+	"github.com/timescale/promscale/pkg/pgmodel/model"
 	"github.com/timescale/promscale/pkg/prompb"
 )
 
-// Labels stores a labels.Labels in its canonical string representation
-type Labels struct {
-	Names      []string
-	Values     []string
-	MetricName string
-	str        string
+const DefaultSeriesCacheSize = 500000
```

Comment:
Can we please have a comment as to how setting this high helps us, based on the recent benchmarks?